templates/ai/content/ai-basics/index-content.html

```diff
@@ -1,4 +1,8 @@
 <h2 id="ai-basics">AI Basics</h2>
+<code>
+{% include "ai/content/ai-basics/_ai-basics-carousel.html" %}
+</code>
+
 <h4 id="what-is-ai-in-2023">What is AI in 2023?</h4>
```

Comment:
Not relevant here, but since the eyes see it (and a11y linting will undoubtedly echo it), it looks like we're jumping from an `<h2>` => `<h4>` here (and presumably elsewhere).
Me thinks these should [eventually] be converted to H3 tags to keep them in sequential descending order.

Reply:
Good call - I changed this (and also the build script, which is looking for h4s. But I think this again points out where the build script is a bit brittle @skyfallsin - we're gonna need more flexibility here to use different level h tags and still be able to generate the needed markup / IDs / etc.

Reply:
Yeah, generally in the past I've used [11ty](https://www.11ty.dev/), with the [markdown-it-anchor](https://www.npmjs.com/package/markdown-it-anchor) plugin, to manage auto-generated `id`s in my heading tags, but then you'd still have to hardcode the anchors from your side nav, so maybe it's moot.
But I think you might be able to simplify a bunch of the plumbing using 11ty.